### PR TITLE
TCM-708 | Implement view video analytics events

### PR DESCRIPTION
**Pull request by devender197**
### Summary

- [x] VideoEvent, ViewVideo and ViewVideoDetail analytics set

### Issue

[TCM-708](https://hyfnla.atlassian.net/browse/TCM-708)

#### Event List
Screen Shot 2021-07-06 at 3 07 59 PM

##### On Demand Video
Screen Shot 2021-07-06 at 3 07 41 PM

##### Home
Screen Shot 2021-07-06 at 3 07 18 PM

### CODE REVIEW

**Review by onodude — CHANGES_REQUESTED** [review]
looks like you over scoped for the sessions not sure why you would add to live and other stuff
this is only about on demand videos please make the necessary adjustments

**Comment by devender197** [comment]
> looks like you over scoped for the sessions not sure why you would add to live and other stuff
> this is only about on demand videos please make the necessary adjustments

[Done](Remove the Live and Virtual screen Video View detail analytics)

File: app/src/main/java/com/trx/consumer/screens/discover/DiscoverViewModel.kt [hunk 1]
```diff
@@ -119,6 +119,7 @@ class DiscoverViewModel @ViewModelInject constructor(
     }
 
     override fun doTapVideo(model: VideoModel) {
+        analyticsManager.trackViewVideoDetail(model, AnalyticsPageModel.DISCOVER)
```
Comment (vespertineSiren) [inline app/src/main/java/com/trx/consumer/screens/discover/DiscoverViewModel.kt:122]: Can you make this look like:
`analyticsManager.trackViewVideoDetail(model, DISCOVER)`

You did something similarly on Line 20 in the `HomeViewModel`.

**Review by vespertineSiren — CHANGES_REQUESTED** [review]
LGTM. Just one small change.

File: app/src/main/java/com/trx/consumer/screens/discover/DiscoverViewModel.kt (hunk 1 above)
Comment (devender197) [reply app/src/main/java/com/trx/consumer/screens/discover/DiscoverViewModel.kt]: [Done](https://github.com/hyfn/trx-consumer-android/pull/159/commits/59f69a2d6b2d752415daf0291507bae730e814b1)

**Review by onodude — APPROVED** [review]
thank you!

**Pull request merged**